frontend/src/views/Collections/index.tsx

```diff
@@ -53,23 +52,10 @@ const COLUMN_ID_RECENCY = "recency";
  */
 const RECENCY = "recency";
 
-/**
- * Gene sets CSV upload functionality, available if gene sets feature flag is enabled
- */
-const AsyncUploadCSV = loadable(
```

Comment:
Removed as per [Slack](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1642655185046100).

Reply:
broken link for CZI folks